tests/xharness/Collections/ILoadAsync.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Threading.Tasks;
+using Xharness;
+using Xharness.Logging;
+
+namespace xharness.Collections{
+	public interface ILoadAsync {
+		Task LoadAsync (ILog log, bool include_locked, bool force);
```

Comment:
>include_locked [](start = 33, length = 14)

Here it says `include_locked` but in the `Devices` implementation it is `remove_locked` and in `Simulators` impl it is `include_locked`.
In Jenkins it is that used as `ILoadAsync` which might then cause different behaviour based on what is inside the `ILoadAsync` pointer

Reply:
cc @rolfbjarne

---
In reply to: [390979607](https://github.com/xamarin/xamarin-macios/pull/8086#discussion_r390979607) [](ancestors = 390979607)

Reply:
I will open an issue to track this and we can merge here